### ISSUE TITLE
Coming soon badge background color

### ISSUE THREAD

**Issue by lsl** [Automattic/wp-calypso#101285]
The coming soon badge shown to new sites is lacking contrast on the background and lacking a background at all on wp-admin screens.

We should:
- [ ] At least match calypso
- [ ] Ideally increase the contrast of these for all color schemes
- [ ] Consider removing entirely or merging the badge into the launch site button
- [ ] Consider hiding the coming soon badge while Launch site button is visible

Note some overlap with https://github.com/Automattic/wp-calypso/issues/101284

<img width=350 src=https://github.com/user-attachments/assets/8ae2e4bc-c609-4995-a92c-ca92b6a0b2f4 />
<img width=350 src=https://github.com/user-attachments/assets/97d97be6-4077-41fc-a828-9ffa48d5fb10 />
<img width=350 src=https://github.com/user-attachments/assets/527cbd59-06a3-4b90-8edc-199a2752495a />
<img width=350 src=https://github.com/user-attachments/assets/37023937-c4c3-4b6a-b510-58c9a4511050 />

**Comment by lsl** [comment]
@Automattic/dotcom-design similar situation to the wp-admin launch site button colors in: https://github.com/Automattic/wp-calypso/issues/101284 but we have a few more options here. Could you advise on the preferred approach?

**Comment by jasmussen** [comment]
CC: @fditrapani as I know you have worked on Premium badges recently. One instinct I had was to just make this badge match other badges, i.e. white text on black. Unfortunately that doesn't work too well:

![Image](https://github.com/user-attachments/assets/7c77687f-0fe2-4647-846d-8416a7114fa8)

Neither does white on black:

![Image](https://github.com/user-attachments/assets/8cc0e8df-a23a-431a-b43d-1df98fa8ba88)

I also wondered, what if it just uses inherited text color, and a semitransparent currentColor for the border?

![Image](https://github.com/user-attachments/assets/08296b54-3fd2-46c6-8e51-88fc3e370e00) 

You could do this using something like:

```
.badge {
	background: transparent;
	color: inherit;
	border-color: color-mix( in srgb, currentColor 50%, transparent );
}
```

But absent that resonating, the best bet is probably to go with your initial instinct: that the chip should have the same colors, always, as the Primary button of the color scheme.

**Comment by crisbusquets** [comment]
> I also wondered, what if it just uses inherited text color, and a semitransparent currentColor for the border?

Looks good, and it doesn't have the same prominence as the "Launch" button... which is something that will improve the IA.

> But absent that resonating, the best bet is probably to go with your initial instinct: that the chip should have the same colors, always, as the Primary button of the color scheme.

This is what is making the IA complex. Everything here is like "hello, look at me first!".

<img width="1621" alt="Image" src="https://github.com/user-attachments/assets/2d6310de-c0c5-4300-9902-955d04f6d68e" />

Clicking the `Coming soon` label only opens a preview, so it's not an action that should be _that_ prominent (unless I'm missing something).

**Comment by arthur791004** [comment]
> I also wondered, what if it just uses inherited text color, and a semitransparent currentColor for the border?

Here are the result that follows this rule:

| Scheme | Coming Soon |
| - | - |
| fresh (default) | <img width="269" alt="Image" src="https://github.com/user-attachments/assets/afc86610-9e81-4453-87b2-a6d2bf662946" /> |
| light | <img width="271" alt="Image" src="https://github.com/user-attachments/assets/446e69a3-c898-40da-af36-e57e8f602c6a" /> |
| modern | <img width="269" alt="Image" src="https://github.com/user-attachments/assets/de27694b-4efe-4b4f-ba8f-db63cb0e70eb" /> |
| blue | <img width="268" alt="Image" src="https://github.com/user-attachments/assets/26fbc9be-2bbf-4bf9-bbb0-28a22d3b609a" /> |
| midnight | <img width="267" alt="Image" src="https://github.com/user-attachments/assets/a6d28c84-87ea-43e2-a927-6958e3cdbbb1" /> |
| sunrise | <img width="267" alt="Image" src="https://github.com/user-attachments/assets/f925127a-5f7a-4aec-98af-02d58c497fb4" /> |
| ectoplasm | <img width="268" alt="Image" src="https://github.com/user-attachments/assets/1955ff65-59ec-49e2-92e4-3a340e08b17c" /> |
| ocean | <img width="267" alt="Image" src="https://github.com/user-attachments/assets/8b6b0b8e-c4b9-47e3-8efc-b7425ab28fd7" /> |
| coffee | <img width="265" alt="Image" src="https://github.com/user-attachments/assets/f91c687d-6e2f-4d2d-b2f4-a3afe22193f2" /> |

All looks good! Do we want to go with this way?

**Comment by jasmussen** [comment]
Yes, that seems the best path forward at this point, unless Filippo or others chime in with differing opinions.

**Comment by crisbusquets** [comment]
Works for me! It'll help with the IA.

**Comment by fditrapani** [comment]
Looks great! Thank you all for the updates.

**Comment by crisbusquets** [comment]
> Shouldn't we be removing this site card widget?

In the future, yes. But I'm not aware of any specific plan 🤔  I think it's ok to do this quick improvement while we decide what to do.

**Comment by jasmussen** [comment]
The benefit of this quick iteration is that at least it's color scheme agnostic, so it does reduce some entropy.

**Comment by youknowriad** [comment]
I'm actually working on removing the site card in #100764 and related wpcom PRs. Unclear when it will land, but I'm iterating on it.

**Comment by arthur791004** [comment]
Closing it via https://github.com/Automattic/jetpack/pull/42496 and https://github.com/Automattic/wp-calypso/pull/101437.
